src/Button/Button.jsx

```diff
@@ -1,9 +1,20 @@
-import './style.css'
-
-export default function Button({ children = "╳", ...delegatedProps }) {
-    return (
-        <button {...delegatedProps}>
-            {children}
-        </button>
-    );
-}
\ No newline at end of file
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+
+const Button = React.forwardRef(({ children = '╳', ...delegatedProps }, ref) => (

```

Comment:
You can remove = '╳' because you already have it in defaultProps

Reply:
Removed it.